### PR TITLE
Linzen task

### PR DESCRIPTION
**Pull request by viking-sudo-rm**
Implemented Linzen agreement task.

The Linzen task now gets 96.7% accuracy after 20 epochs. I disabled our custom initialization in favor of the default initialization, and it increased performance. This could explain the odd LSTM behavior that we have observed more generally.

This is still not replicating the reported results, however. The other main architectural difference as far as I can tell is the softmax versus sigmoid loss function.

### CODE REVIEW

**Comment by viking-sudo-rm** [comment]
Note: initialization might be slightly different between Keras LSTM and Pytorch.

See:
https://faroit.github.io/keras-docs/1.2.1/layers/recurrent/#lstm
https://pytorch.org/docs/stable/nn.html
